lib/lolcommits/capturer.rb

```diff
@@ -14,3 +14,11 @@ def initialize(attributes = {})
     end
   end
 end
+
```

Comment:
My initial thought was that since those requires are classes which extend the base Capturer class, they would have to be loaded afterwards so the class they extended was known about, but perhaps Ruby doesn't do things in an order specific way like that?